src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -17,34 +17,32 @@ import scala.util.{Failure, Success}
 /**
   * Entry point for synchronizing data between the Tezos blockchain and the Conseil database.
   */
-object Lorre extends App with TezosErrors with LazyLogging {
+object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
 
-  //keep this import here to make it evident where we spawn our async code
+  //reads all configuration upstart, will only complete if all values are found
+  val config = loadApplicationConfiguration(args)
+
+  //stop if conf is not available
+  config.left.foreach { _ => sys.exit(1) }
```

Comment:
Once again, should the error output to the logs be written here instead of inside?

Reply:
The same reasoning exposed in the analogous comment applies here, please let me know if you see any strong argument against it